### PR TITLE
Use Jest

### PR DESCRIPTION
**Pull request by kendaleiv**

### CODE REVIEW

File: tests/analyzers/statistics.test.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  const applicableMessages = result.messages
+    .filter(x => x.type !== 'statistics-text-length');
+
+  // console.log(applicableMessages.map(x => x.type
+  //   + ' : '
+  //   + x.text
+  //   + ' : '
+  //   + initialValues.messages.find(y => y.type === x.type).text));
+
```
Comment (comfroels) [inline tests/analyzers/statistics.test.js]: Were you leaving this in for a reason?

File: tests/validator.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  const result = Validator.isValidMessageType(messageType);
+
+  expect(result).toBe(false);
+});
+
+test('ensureValidMessageType should not throw when valid', () => {
+  const messageType = 'valid-message-1';
+
+  Validator.ensureValidMessageType(messageType);
```
Comment (comfroels) [inline tests/validator.test.js]: Does this test not contain any type of assertion? Will the tests fail if that method fails?

Comment (kendaleiv) [inline tests/validator.test.js]: I expect this to be equivalent to

```javascript
expect(Validator.ensureValidMessageType(messageType)).not.toThrow();
```

However, that doesn't work in this version of Jest.

File: tests/analyzers/statistics.test.js (hunk 1 above)
Comment (kendaleiv) [inline tests/analyzers/statistics.test.js]: Removed in 4f36a98.

File: tests/validator.test.js (hunk 2 above)
Comment (comfroels) [inline tests/validator.test.js]: Okay gotcha

**Pull request merged**